frontend/src/app/shared/slack-invite-box/slack-invite-box.widget.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
This file might cause a merge conflict - I think this is already merged in.

Reply:
Nevermind, it was not! Again, as I mentioned, this PR contains too many new features, so it would probably be good to move this out of the PR for UTA application so it can be reviewed separately later